Doc/library/array.rst

```diff
@@ -60,6 +60,9 @@ Notes:
    .. deprecated-removed:: 3.3 3.16
       Please migrate to ``'w'`` typecode.
 
+(2)
+   As of 2024, 'h', 'i', 'l', and 'q' are usually 2, 4, 4 on Windows else 8, and 8 bytes.
```

Comment:
This note talks about `h`, so if it's kept, the `\(2)` should be added to the `h` & `H` lines in the table.

But, maybe this PR should be closed without merging. If you're wrinting cross-platform code you shouldn't rely on “common” values; if not you can use `itemsize` to find values that are actually right for your platform.